Tests/test_generator.py

```diff
@@ -351,7 +351,7 @@ def lstate(size):
         func = """
 def fetest(%s):        
     ret = 0
-    for i in range(%i):
+    for i in xrange(%i):
```

Comment:
I will fix this one since it was correct before and I inadvertently changed it. Note that I noticed quite a number of other similar cases (code inside a string) which are not valid Python 3 (a lot of print statements). I didn't touch them since the goal of this PR was just to bring in all the testing fixes/updates.